flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/source/config/SourceConfiguration.java

```diff
@@ -203,12 +175,7 @@ public boolean isEnableMetrics() {
 
     /** Convert the subscription into a readable str. */
     public String getSubscriptionDesc() {
-        return getSubscriptionName()
-                + "("
-                + getSubscriptionType()
-                + ","
-                + getSubscriptionMode()
-                + ")";
+        return getSubscriptionName() + "(" + getSubscriptionMode() + ")";
```

Comment:
Maybe we hard code the type as `Exclusive`? I think it's still expressive and meaningful.